lib/update.php

```diff
@@ -70,6 +70,9 @@ private function _hooks() {
 
     // Add activation key message for plugins with missing keys
     add_action( 'load-plugins.php', array( $this, 'vimeography_check_for_missing_activation_keys' ) );
+    
+    // Add activation expiration message for plugins with expiring keys 
+    add_action( 'wp_loaded', array( $this, 'vimeography_check_for_expiring_keys' ) );
```

Comment:
Suggest `admin_init` hook instead of `wp_loaded` since we're only showing messages on admin.